packages/azure-kusto-ingest/src/resourceManager.ts

```diff
@@ -252,6 +256,20 @@ export class ResourceManager {
         return this.refreshAuthorizationContext();
     }
 
+    async getBlockBlobClient(blobName: string) {
+        const containers = await this.getContainers();
+        if (containers == null) {
+            throw new Error("Failed to get containers");
+        }
+        const container = containers[Math.floor(Math.random() * containers.length)];
+        const containerClient = new ContainerClient(container.uri);
+        return containerClient.getBlockBlobClient(blobName);
+    }
+
+    async getStatusTable() {
```

Comment:
Should maybe already call createStatusTableClient?

Reply:
I prefer not - all the other getXX return ResourceURI

Reply:
also the uri itself is used also without the client

Reply:
Then have a convenience method inside the client?